nav.go

```diff
@@ -525,6 +525,13 @@ func (nav *nav) checkDir(dir *dir) {
 		nav.startPreview()
 		dir.loading = true
 		dir.loadTime = now
+
+		// Cryfs filesystems always show as having 0 blocks and are always considered to be modified at current time
+		blockSize, err := getBlockSize(s)
+		if err == nil && *blockSize == 0 {
+			return
+		}
+
```

Comment:
This code should be moved to before the `nav.startPreview()` line, otherwise `lf` will think the directory is going to be loaded, and will display `loading...` in the UI (if the mount directory is empty), see the following code: https://github.com/gokcehan/lf/blob/c88b8eaa6544824bde136bf7d60b5d571e8ccd27/ui.go#L362-L367

Reply:
Moved up.